pkg/storage/postgres/postgres.go

```diff
@@ -99,8 +103,8 @@ func New(uri string, cfg *sqlcommon.Config) (*Postgres, error) {
 	policy := backoff.NewExponentialBackOff()
 	policy.MaxElapsedTime = 1 * time.Minute
 	attempt := 1
-	err = backoff.Retry(func() error {
-		err = db.PingContext(context.Background())
+	err := backoff.Retry(func() error {
+		err := db.PingContext(context.Background())
```

Comment:
I'm curious, why this change, what does it improve?

Reply:
Hi, this change improves nothing but to make sure the code can actually compile because this is the first declaration of the variable `err` in the block (after breaking into two functions).

Reply:
Oh right haha. Thanks